### DISCUSSION TITLE
v0.3.12

### DISCUSSION THREAD

**Discussion by dynobo** [dynobo/normcap#270] [Announcements]
See [CHANGELOG](https://github.com/dynobo/normcap/blob/main/CHANGELOG.md) for details.

<hr /><em>This discussion was created from the release <a href='https://github.com/dynobo/normcap/releases/tag/v0.3.12'>v0.3.12</a>.</em>

**Comment by SpecterShell** [comment]
The date in the CHANGELOG is wrong.
https://github.com/dynobo/normcap/commit/b9114e195c189cee4a1e7f19ccbd225a3137d081#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL3

**Reply by dynobo** [reply]
Thanks! Fixing it right away :+1:

**Reply by dynobo** [reply]
Commit: https://github.com/dynobo/normcap/pull/274/commits/b1f36d126bdfbe95aef382dc75d2a6234ed4c74d
